dist/setup/index.js

```diff
@@ -101717,21 +101717,24 @@ class MicrosoftDistributions extends base_installer_1.JavaBase {
             // TODO get these dynamically!
             // We will need Microsoft to add an endpoint where we can query for versions.
             const jdkVersions = [
+                {
+                    version: [17, 0, 3]
```

Comment:
Should 17.0.3 not replace 17.0.1.12.1?

Does 17.0.3 need to have the full build number written out?

Reply:
It does replace it as the latest. We still preserve `17.0.1.12.1` for history though.

No the version number changed recently to exclude the build number